src/pages/myNotes/componentInfo.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
```

Comment:
remove space here, try to group all external dependencies together

Reply:
Removed